Documentation/teaching/so2/assign0-kernel-api.rst

```diff
@@ -0,0 +1,120 @@
+=========================
```

Comment:
Please provide in the commit message a link to the initial assignment statement on OCW. 
Also, in the commit message we should see a brief description about what is all about.

e.g

```
Add SO2 assignment 0 to git repo. Original is at: <insert link>

This assignment asks the students to implement a simple Linux kernel module to demonstrate the Linux kernel list API.
```

Reply:
Fixed.